source/world/entity/MobFactory.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "EntityType.hpp"
+#include "Mob.hpp"
+
+class MobFactory
+{
+	static Mob* CreateMob(EntityType entityType, Level *level);
```

Comment:
Why not use camelCase like the rest of ReMCPE?

Reply:
A. Mojang did it like this.
B. These are static functions so I would imagine the naming convention would make sense in this case.